src/aiida/orm/nodes/data/data.py

```
@@ -42,14 +40,14 @@ class Data(Node):
     # By default, if not found here,
     # The fileformat string is assumed to match the extension.
     # Example: {'dat': 'dat_multicolumn'}
-    _export_format_replacements: Dict[str, str] = {}
+    _export_format_replacements: dict[str, str] = {}
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)